sqlitecluster/SQLiteClusterMessenger.cpp

```diff
@@ -241,6 +241,65 @@ unique_ptr<SHTTPSManager::Socket> SQLiteClusterMessenger::_getSocketForAddress(s
     return s;
 }
 
+bool SQLiteClusterMessenger::runOnValidFollowerPeer(BedrockCommand& command) {
```

Comment:
This function is almost identical to `runOnLeader`. We should refactor out all of the common functionality so we don't duplicate it.

Reply:
Agreed, refactored